prepare/metrics/rag.py

```diff
@@ -39,7 +31,7 @@
     {"f1": 0, "precision": 0, "recall": 0, "score": 0, "score_name": "f1"},
 ]
 
-# Currently CopyFields does not delete the source fields,
+# Currently, CopyFields does not delete the source fields,
```

Comment:
will RenameFields  fit better here?

Reply:
I'm not familiar with this, @elronbandel - what do you think?

Reply:
If you need 'move' rather than 'copy', then 'RenameField' will do the job for you. It carefully removes the source, paying attention to dpath subtle situations.